float32 odometryX     # X position
float32 odometryY     # Y position
bool bumperW     # left bumper
bool bumperN    # front bumper
bool bumperE    # right bumper
uint32 power # power system state: 0: unplugged, 1: plugged to adapter, 2: plugged to dock, 3: charged, 4: battery_low, 5: battery_critical 
